### PR TITLE
{system}[GCCcore/11.2.0] strace v5.14

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
jsfc01.int.jusuf.sebastian.cluster - Linux rocky linux 8.4, x86_64, AMD EPYC 7742 64-Core Processor (zen2), 1 x NVIDIA GRID V100-4C, 460.73.01, Python 3.6.8
See https://gist.github.com/5b1c7e3f817d2ca1b855757b71d1d453 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
rocky8-eb - Linux rocky linux 8.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.8
See https://gist.github.com/fca504644721d7f6a3a3d4d933d1faa9 for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/59029b056f404f6671bc802c1f1ee38e for a full test report.

File: easybuild/easyconfigs/s/strace/strace-5.14-GCCcore-11.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+checksums = ['901bee6db5e17debad4530dd9ffb4dc9a96c4a656edbe1c3141b7cb307b11e73']
+
+builddependencies = [
+    ('binutils', '2.37'),
+]
+
+sanity_check_paths = {
+    'files': ['bin/strace-log-merge', 'bin/strace'],
+    'dirs': ['share']
+}
```
Comment (bedroge) [inline easybuild/easyconfigs/s/strace/strace-5.14-GCCcore-11.2.0.eb:26]: Could you add a sanity check command here, e.g. `strace --help`?
```suggestion
sanity_check_paths = {
    'files': ['bin/strace-log-merge', 'bin/strace'],
    'dirs': ['share']
}

sanity_check_commands = ['strace --help']
```

**Review by bedroge — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/strace/strace-5.14-GCCcore-11.2.0.eb (hunk 1 above)
Comment (bedroge) [reply easybuild/easyconfigs/s/strace/strace-5.14-GCCcore-11.2.0.eb:26]: Committing this, as it's a minor change/improvement.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 20.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.8.10
See https://gist.github.com/a4d5d828b8198b2e8160faaf5e7ef09d for a full test report.

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/ffc4b1dc0140464492eb4f5c8f191fda for a full test report.

**Comment by bedroge** [comment]
@SebastianAchilles I'm getting the following error on an Ubuntu system:

```
checking for m32 personality compile support (using gcc -I/data/apps/software/binutils/2.37-GCCcore-11.2.0/include -I./bundled/linux/arch/x86/include/uapi -I./bundled/linux/include/uapi -Isrc  -m32)... no
checking whether to enable m32 personality support... no
configure: error: Cannot enable m32 personality support
```

https://www.linuxquestions.org/questions/slackware-14/regression-on-current-with-binutils-2-36-a-4175689373/

Adding `--enable-mpers=check` does solve the issue, but I'm not entirely sure what that does / if we need this functionality?

**Comment by boegel** [comment]
> @SebastianAchilles I'm getting the following error on an Ubuntu system:
> 
> ```
> checking for m32 personality compile support (using gcc -I/data/apps/software/binutils/2.37-GCCcore-11.2.0/include -I./bundled/linux/arch/x86/include/uapi -I./bundled/linux/include/uapi -Isrc  -m32)... no
> checking whether to enable m32 personality support... no
> configure: error: Cannot enable m32 personality support
> ```
> 
> https://www.linuxquestions.org/questions/slackware-14/regression-on-current-with-binutils-2-36-a-4175689373/
> 
> Adding `--enable-mpers=check` does solve the issue, but I'm not entirely sure what that does / if we need this functionality?

That seems like it will conditionally build support for the `mpers` feature when it's possible?

Seems like a reasonable thing to include in the easyconfig...

**Comment by boegel** [comment]
@SebastianAchilles ping?

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-ubuntu-2204 - Linux Ubuntu 22.04.2 LTS (Jammy Jellyfish), x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.6
See https://gist.github.com/SebastianAchilles/4f3f96b382056556ef72efba398472b9 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-opensuse-155 - Linux openSUSE Leap 15.5, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.15
See https://gist.github.com/SebastianAchilles/d43ff4fb4574339829bd2e7c6bb8b21f for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bwd-rockylinux-92 - Linux Rocky Linux 9.2 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.16
See https://gist.github.com/SebastianAchilles/40fd037ac731352c90c1e8f57e5e855d for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
zen2-rockylinux-88 - Linux Rocky Linux 8.8, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/1c6924416a74d8cc821b4082083321ec for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
skl-centos-79 - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.6.8
See https://gist.github.com/SebastianAchilles/5377cfd4e993d8d970ddf5af0ec481c9 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/bedroge/e1cc8437b5cbc9b4245aa8a4a658ff5e for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @SebastianAchilles!

**Pull request merged**